settings.gradle.kts

```diff
@@ -122,15 +127,13 @@ dependencyResolutionManagement {
         // distribution. These libs can be depended on during compilation, or bundled as part of
         // runtime.
         create("libs") {
-            // The HAPI API version to use, this need to match the tag set on gitRepositories above
-            val hapiVersion = "0.40.0-blocks-state-SNAPSHOT"
-
             val besuNativeVersion = "0.6.1"
             val besuVersion = "23.1.2"
             val bouncycastleVersion = "1.70"
             val daggerVersion = "2.42"
             val eclipseCollectionsVersion = "10.4.0"
             val helidonVersion = "3.0.2"
+            val ioGrpcVersion = "1.45.1"
```

Comment:
I think it's better to keep the `grpc.*` modules versions consistent and at the state that is currently on develop. I changed this back, because there were failing tests and I was not sure what the cause is.

I would prefer to do the upgrade in a separate PR to get this one merged now.